ci/Jenkinsfile

```diff
@@ -1,5 +1,7 @@
 node(label: 'linux') {
-  sh 'npm run docker-build'
-  sh 'npm run docker-test'
-  sh 'archiveArtifacts artifacts: 'build/*.zip''
+  checkout scm
+  sh 'docker build -t ipfs-companion .'
+  sh 'docker run -i ipfs-companion yarn test'
```

Comment:
Note to self: we may want to move tests after build when we have integration ones that run in real browser, but for now its fine as is 👍

Reply:
Hm, I don't think so, we want to make sure that the application is correct before building it. A alternative solution could be to do unit tests + building sequentially in one pipeline and then run functional tests in it's own. If the functional tests fails, stop the other pipeline. 

We need to make some basic assertions before building though.